src/components/AdminSettings.vue

```diff
@@ -59,8 +68,12 @@
 						<div v-if="!state.speech_to_text_picker_available" class="checkbox-text">
 							<InformationOutlineIcon class="icon" />
 							<span>
-								{{ t('assistant', 'To enable this feature, please install a speech-to-text provider.') }}
+								{{ t('assistant', 'To enable this feature, please install a speech-to-text provider:') }}
```

Comment:
```suggestion
								{{ t('assistant', 'To enable this feature, please install a speech-to-text provider') + ':' }}
```

Reply:
huh. Not sure if this will have the intended effect in all languages (ie. RTL?)